src/full/Agda/TypeChecking/InstanceArguments.hs

```diff
@@ -548,13 +553,14 @@ shouldPostponeInstanceSearch =
 nowConsideringInstance :: (ReadTCState m) => m a -> m a
 nowConsideringInstance = locallyTCState stConsideringInstance $ const True
 
+isInstanceProblemConstraint :: ProblemConstraint -> Bool
```

Comment:
Pulled this out of the `where` clause for `wakeupInstanceConstraints` below, since I needed it for the `holdConstraints` call above.